### ISSUE TITLE
Cascade Delete does not trigger realtime callback

### ISSUE THREAD

**Issue by odama626** [supabase/supabase#6739]
# Bug report

## Describe the bug
realtime subscription '*' does not trigger when a record is deleted by cascade

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to '…'
2. Click on '…'
3. Scroll down to '…'

## Expected behavior
the callback should be called

## System information

- OS: [e.g. macOS, Windows] Manjaro
- Browser (if applies) [e.g. chrome, safari] chrome
- Version of supabase-js: [e.g. 6.0.2] latest docker stack
- Version of Node.js: [e.g. 10.10.0] 16.11.1

## Additional context
I'm subscribed to all updates on a join table, when invoking a delete that cascade deletes a record in the join table does not trigger the callback

**Comment by kiwicopple** [comment]
hey @odama626 - can I doublecheck that you have "Delete" enabled for the replication slot (Under Database > Replication > `supabase_realtime`)?

<img width="1405" alt="image" src="https://user-images.githubusercontent.com/10214025/169772442-fa04dedd-0f39-4638-bed8-0ea1bc96a112.png">

**Comment by GaryAustin1** [comment]
I set up a test case of realtime on a table2 with a foreign key with cascade delete pointing to table1.
Realtime reported a delete event when I deleted the row in table 1, showing the row in table 2 being deleted.

If this is an issue, it is likely a special case for the OP or has been fixed.

I'm going to close this issue.  If the OP still has problems I suggest a discussion here in github or in Supabase Discord server to try and narrow the problem.
Thanks
